images/metering-ansible-operator/roles/meteringconfig/tasks/configure_tls.yml

```diff
@@ -1,7 +1,21 @@
 ---
 
+- name: Create temporary directory to store all the necessary certificates/keys
+  tempfile:
+    suffix: certificates
+    state: directory
+  register: certificates_dir
+
+- name: Generate root certificate authority
+  include_tasks: configure_root_ca.yml
+
 - name: Configure TLS and client-side authentication in Presto
   include_tasks: configure_presto_tls.yml
 
 - name: Configure TLS and authentication in the reporting-operator
   include_tasks: configure_reporting_operator_tls.yml
+
+- name: Cleanup the temporary directory which held the certificates and keys
```

Comment:
This will be skipped if the previous tasks fail. The other includes need to be in a block with this as the `always:`

Reply:
good catch